modules/calib3d/test/test_homography.cpp

```diff
@@ -711,4 +712,29 @@ TEST(Calib3d_Homography, fromImages)
     ASSERT_GE(ninliers1, 80);
 }
 
+TEST(Calib3d_Homography, minPoints)
```

Comment:
I guess a supplementary test with input points < 4 and check that an exception is correctly thrown should be added?